audit/format.go

```diff
@@ -434,3 +434,25 @@ func parseVaultTokenFromJWT(token string) *string {
 
 	return &claims.ID
 }
+
+// Create a formatter not backed by a persistent salt.
+func NewTemporaryFormatter(format string, prefix string) *AuditFormatter {
```

Comment:
```suggestion
func NewTemporaryFormatter(format, prefix string) *AuditFormatter {
```

total nit, but the go-y thing to do